Assets/HoloToolkit/Interaction/Scripts/Receivers/ToggleActiveReceiver.cs

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+//
+using UnityEngine;
+using System.Collections;
+using System;
+using HoloToolkit.Unity.InputModule;
+
+namespace HoloToolkit.Unity.Receivers
+{
+    /// <summary>
+    /// Simple receiver class for toggling a game object active or inactive.
+    /// </summary>
+    public class ToggleActiveReceiver : InteractionReceiver
+    {
+		/// <summary>
+        /// When receiving the input down button toggle active/inactive all targets
+        /// </summary>
+        /// <param name="obj"></param>
+        /// <param name="args"></param>
+		protected override void InputDown(GameObject obj, InputEventData args)
+		{
+            Debug.Log("On tapped in interaction receiver");
```

Comment:
Could we make this optional?

Reply:
Hi @StephenHodgson,

Which part would you want to be optional?  The debug log?  This is more a simple example script to show easy usage for InteractionReceiver and the debug log could be removed completely.  Do we define anywhere a conditional for logging?  Might be something to add in the future to enable/disable log spew.

thanks,
-pat

Reply:
Yeah just the debug log.

Reply:
Yeah totally agree and usually have a #define ENABLE_LOGGING or something equivalent for toggling on and off.

thanks,
-pat